example.txt

```diff
@@ -0,0 +1,7 @@
+for(int i = 0; i < 10; i++){

```

Comment:
This file should be .cpp extension

Reply:
Oh, can Go handle that? I didn't know we could read other file extensions.

Reply:
Yeah, Go can read all file types (they're just treated as byte streams with funny names.)